runtime/oti/jvminit.h

```diff
@@ -512,6 +512,8 @@ enum INIT_STAGE {
 #define MAPOPT_XXONOUTOFMEMORYERROR_EQUALS "-XX:OnOutOfMemoryError="
 #define MAPOPT_XXENABLEEXITONOUTOFMEMORYERROR "-XX:+ExitOnOutOfMemoryError"
 #define MAPOPT_XXDISABLEEXITONOUTOFMEMORYERROR "-XX:-ExitOnOutOfMemoryError"
+#define MAPOPT_XXLARGEPAGESIZEINBYTES_EQUALS "-XX:LargePageSizeInBytes="
+#define MAPOPT_XXUSELARGEPAGES "-XX:+UseLargePages"
```

Comment:
Should it be recognition of `"-XX:-UseLargePages"` for symmetry ?

Reply:
@pshipton Should this be added? 

In theory, it makes sense to have an option to disable all large page options, including -Xlp*. However this PR is to map Hotspot Options, and Hotspot does not support such an option.

Reply:
Yes, this needs to have the `-XX:+UseLargePages` to be consistent with our standard processes.  If there is a `+` version, there *must* be a `-` version as well.  Otherwise, use the non-[+-] `-XX:Foo` style option